var/spack/repos/builtin/packages/boxlib/package.py

```diff
@@ -25,26 +25,28 @@
 from spack import *
 
 
-class Boxlib(Package):
+class Boxlib(CMakePackage):
     """BoxLib, a software framework for massively parallel
        block-structured adaptive mesh refinement (AMR) codes."""
 
     homepage = "https://ccse.lbl.gov/BoxLib/"
-    url = "https://ccse.lbl.gov/pub/Downloads/BoxLib.git"
+    url = "https://github.com/BoxLib-Codes/BoxLib/archive/16.12.2.tar.gz"
 
-    # TODO: figure out how best to version this.  No tags in the repo!
-    version('master', git='https://ccse.lbl.gov/pub/Downloads/BoxLib.git')
+    version('16.12.2', 'a28d92a5ff3fbbdbbd0a776a59f18526')
 
     depends_on('mpi')
-    depends_on('cmake', type='build')
-
-    def install(self, spec, prefix):
-        args = std_cmake_args
-        args += ['-DCCSE_ENABLE_MPI=1',
-                 '-DCMAKE_C_COMPILER=%s' % which('mpicc'),
-                 '-DCMAKE_CXX_COMPILER=%s' % which('mpicxx'),
-                 '-DCMAKE_Fortran_COMPILER=%s' % which('mpif90')]
-
-        cmake('.', *args)
-        make()
-        make("install")
+
+    def cmake_args(self):
+        spec = self.spec
+        options = []
+
+        options.extend([
+            # '-DBL_SPACEDIM=3',
+            '-DENABLE_POSITION_INDEPENDENT_CODE=ON',
+            '-DENABLE_FBASELIB=ON',
+            '-DCMAKE_C_COMPILER=%s' % spec['mpi'].mpicc,
+            '-DCMAKE_CXX_COMPILER=%s' % spec['mpi'].mpicxx,
+            '-DCMAKE_Fortran_COMPILER=%s' % spec['mpi'].mpifc
+        ])
```

Comment:
Instead of doing:
```python
options = []

options.extend([
    ...
])
```
why not do:
```python
options = [
    ...
]
```
or even:
```python
return [
    ...
]
```